Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM resin/armv7hf-debian:jessie
+
+RUN apt-get update \
+	&& apt-get install -y axel wget htop nano libjpeg-dev libconfig++-dev\
```

Comment:
After the comments bellow, you only need to install `bison flex freebsd-glue libconfig++-dev` and maybe add `htop nano` if you want a debug build